src/me/mrCookieSlime/Slimefun/listeners/ItemListener.java

```diff
@@ -98,9 +98,34 @@ else if (SlimefunManager.isItemSimiliar(slot1, SlimefunGuide.getItem(BookDesign.
 		}
 	}
 
+	/*
+	 * Handles Left click use and checks for disabled items.
+	 */
+	@EventHandler
+	public void enabledCheck(PlayerInteractEvent e) {
+		if(e.getAction() == Action.RIGHT_CLICK_AIR || e.getAction() == Action.RIGHT_CLICK_BLOCK)
```

Comment:
Don't forget Action.PHYSICAL for Pressure Plates, we don't need that triggered.
You should instead check for LEFT_CLICK Actions instead (Just in case more Actions get added)